back-end/src/sciler/handler/handler.go

```diff
@@ -50,6 +50,7 @@ func (handler *Handler) msgMapper(raw Message) {
 		{
 			handler.updateStatus(raw)
 			handler.sendStatus(raw.DeviceID)
+			handler.sendFrontEndStatus(raw)
```

Comment:
can sendStatus, sendFrontEndStatus and sendEventStatus be combined to sendStatusToFrontEnd

Reply:
no we want to split out the functionality